elements/lisk-dpos/src/dpos.ts

```diff
@@ -41,13 +42,15 @@ import {
 	ForgersList,
 	StateStore,
 	DecodedVoteWeights,
-	DecodedUsernames,
 } from './types';
 
 interface DposConstructor {
 	readonly chain: Chain;
+	readonly initDelegates: ReadonlyArray<Buffer>;
 	readonly activeDelegates?: number;
 	readonly standbyDelegates?: number;
+	readonly genesisHeight?: number;
+	readonly initRound?: number;
```

Comment:
```suggestion
	readonly initRounds: number;
```
It should not be optional, must be provided to initiate DPOS.